spec/presenters/individual_statistics_presenter_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require_relative '../../app/presenters/individual_statistics_presenter'
+
+describe IndividualStatisticsPresenter do
+  describe 'articles view count ' do
+    let(:course1) { create(:course) }
+    let(:course2) { create(:course) }
+    let(:user) { create(:user) }
+    let(:article) { create(:article) }
+    subject { described_class.new(user: user) }
+
+    context 'Select unique articles' do
```

Comment:
The conventional way to name the context block is so that it combine with the rest of the spec to form something like a complete sentence, like "IndividualStatisticsPresenter articles_view_count, when a user is in two courses that overlap, doesn't double count the same article in multiple courses".

(If you create `.rspec` in the project root directory and add formatting options, you'll see each of these messages as the tests run. Mine looks like this:
```
--color
--format documentation
```
)